apps/src/lib/ui/simpleSignUp/workshop/WorkshopContinueAccountCard.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import {Button, buttonColors} from '@cdo/apps/componentLibrary/button';
+import {
+  Card,
+  CardActions,
+  CardContent,
+  CardHeader,
+} from '@cdo/apps/componentLibrary/card';
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
+import {navigateToHref} from '@cdo/apps/utils';
+import i18n from '@cdo/locale';
+
+import styles from '../link-account.module.scss';
+
+const WorkshopContinueAccountCard: React.FunctionComponent<{
+  continueAccountUrl: string;
+}> = ({continueAccountUrl}) => (
+  <Card data-testid={'continue-account-card'}>
+    <CardHeader
+      title={i18n.ltiLinkAccountNewAccountCardHeaderLabel()}
+      icon={
+        <FontAwesomeV6Icon
+          className={classNames(styles.cardIcon, 'fa-2x')}
+          iconName={'user-plus'}
+        />
+      }
+    />
+    <CardContent className={styles.cardContent}>
+      {i18n.ltiLinkAccountContinueAccountCardContent()}
+    </CardContent>
+    <CardActions>
+      <Button
+        className={classNames(styles.button, styles.cardSecondaryButton)}
+        color={buttonColors.white}
+        size="m"
+        text={i18n.ltiIframeCallToAction()}
+        onClick={() => navigateToHref(continueAccountUrl)}
+      />
```

Comment:
I think the `<LinkButton>` option would be better here for accessibility if this is navigating away from the page.

`import {LinkButton} from '@cdo/apps/componentLibrary/button';`

```suggestion
      <LinkButton
        color="black"
        size="m"
        text={i18n.ltiIframeCallToAction()}
        type="secondary"
        onClick={() => navigateToHref(continueAccountUrl)}
      />
```

Reply:
@TurnerRiley Erin Bond had a good point in [this comment](https://github.com/code-dot-org/code-dot-org/pull/59924#discussion_r1687241002) re: using `buttonColors.black` (etc) instead of the color name by itself. Sorry for misleading you here!

Reply:
Oh gotcha thank you! On it!

Reply:
Thanks for updating!